### PR TITLE
Fix in PasswordPolicyMaxLogin check

### PR DESCRIPTION
**Pull request by YaaraVerner**
fix edge-case in PasswordPolicyMaxLogin check return UNKNOWN if max_login_attempts is variable.
add a resource to tests for unknown result

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: checkov/terraform/checks/resource/alicloud/PasswordPolicyMaxLogin.py [hunk 1]
```diff
@@ -28,6 +28,8 @@ def scan_resource_conf(self, conf):
 
         if conf.get('max_login_attempts'):
             length = force_int(conf.get('max_login_attempts')[0])
+            if not length:
```
Comment (gruebel) [inline checkov/terraform/checks/resource/alicloud/PasswordPolicyMaxLogin.py:31]:
```suggestion
            if length is None:
```
because `0` is a valid value 🙂

**Review by gruebel — COMMENTED** [review]
nice 👍
